independent-publisher-connectors/OpenAI GPT-4/apiDefinition.swagger.json

```diff
@@ -0,0 +1,652 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "OpenAI",

```

Comment:
@troystaylor it is not allowed to have two connectors with the same name. is it possible to change the name? maybe to "OpenAI GPT-4"